domains/glitched-da-kitty-cat.json

```diff
@@ -0,0 +1,10 @@
+{
+  "owner": {
+    "username": "Glitched Da Kitty Cat",
+    "email": "glitcheddacat@gmail.com",
+    "discord": "glitchedtahcat"
+  },
+  "record": {
+    "TXT": "vc-domain-verify=glitched-da-kitty-cat.is-a.dev,d3ead0cedf604d462fbb"
```

Comment:
Invalid. This vercel verification record should be under a _vercel.domain-request.json file, while another file should have the domain you're trying to request with either A or CNAME. See docs for more information on how to setup a vercel site. https://docs.is-a.dev/guides/vercel